caddyfile/parse.go

```diff
@@ -76,6 +70,7 @@ func (p *parser) begin() error {
 	}
 
 	err := p.addresses()
+
```

Comment:
I'm going to be pedantic here and suggest that this empty line shouldn't be here, as the error check on the next line is directly associated with this line. ;)

Anyway, I won't make this a stopper for the PR if it's the only change I would want.